runners/genlayer-py-std/src/genlayer/chain.py

```
@@ -1,4 +1,4 @@
-__all__ = ('Account', 'Event', 'id')
+__all__ = ('ON', 'IAccount', 'Account', 'Event', 'id')
```

Ruff (0.15.9)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)